examples/official-site/sqlpage/migrations/33_carousel_component.sql

```diff
@@ -0,0 +1,150 @@
+INSERT INTO component (name, description, icon, introduced_in_version)
+VALUES (
+        'carousel',
+        'A carousel is used to display multiple pieces of visual content without taking up too much space.',
+        'carousel-horizontal',
+        '0.18.0'
+    );
+
+INSERT INTO parameter (
+        component,
+        name,
+        description,
+        type,
+        top_level,
+        optional
+    )
+VALUES (
+        'carousel',
+        'name',
+        'Name of the carousel.',
+        'TEXT',
+        TRUE,
+        FALSE
+    ),
+    (
+        'carousel',
+        'title',
+        'Title of the carousel.',
+        'TEXT',
+        TRUE,
+        TRUE
+    ),
+    (
+        'carousel',
+        'indicators',
+        'Style of image indicators (e.g. square, dot).',
+        'TEXT',
+        TRUE,
+        TRUE
+    ),
+    (
+        'carousel',
+        'vertical',
+        'Whether to use the vertical image indicators.',
+        'BOOLEAN',
+        TRUE,
+        TRUE
+    ),
+    (
+        'carousel',
+        'controls',
+        'Whether to show the control links to go previous or next item.',
+        'BOOLEAN',
+        TRUE,
+        TRUE
+    ),
+    (
+        'carousel',
+        'width',
+        'Width of the component, between 1 and 12.',
+        'NUMBER',
+        TRUE,
+        FALSE
+    ),
+    (
+        'carousel',
+        'center',
+        'Whether to center the carousel.',
+        'BOOLEAN',
+        TRUE,
+        TRUE
+    ),
+    (
+        'carousel',
+        'fade',
+        'Whether to apply the fading effect.',
+        'BOOLEAN',
+        TRUE,
+        TRUE
+    ),
+    (
+        'carousel',
+        'image',
+        'The URL (absolute or relative) of an image to display in the carousel.',
+        'URL',
+        FALSE,
+        FALSE
+    ),
+    (
+        'carousel',
+        'title',
+        'Add caption to the slide.',
+        'TEXT',
+        FALSE,
+        TRUE
+    ),
+    (
+        'carousel',
+        'contents',
```

Comment:
Can we call this description to be consistent with other components ?

Reply:
You're right. I think that "description" is more consistent.